packages/typescript/README.md

```diff
@@ -1,8 +1,8 @@
-# Webpack blocks - SASS
+# Webpack blocks - Typescript
 
 [![JavaScript Style Guide](https://img.shields.io/badge/code%20style-standard-brightgreen.svg)](http://standardjs.com/)
 
-This is the `typescript` block providing Typescript support for webpack. Uses `awesome-typescript-loader`.
+This is the `typescript` block providing typescript support for webpack. Uses `awesome-typescript-loader`.
```

Comment:
Just one suggestion: I think it's written "TypeScript" (https://en.wikipedia.org/wiki/TypeScript). But it's really no big deal 😉